### ISSUE TITLE
[Bug] reset_num_timesteps=False doesn't preserve the training curve

### ISSUE THREAD

**Issue by Dylan-Kerler** [DLR-RM/stable-baselines3#433]
### 🐛 Bug

As per the [docs](https://stable-baselines3.readthedocs.io/en/master/guide/tensorboard.html?highlight=reset_num_timesteps#basic-usage), passing the `reset_num_timesteps=False` should preserve the training curve in Tensorboard. Instead, what happens is that a new training curve is created with the same name each time:

![Screenshot 2021-05-15 at 11 41 55](https://user-images.githubusercontent.com/37438950/118357579-be632580-b572-11eb-9552-22eea095b2bf.jpg)

See here that the training curve is starting from 0 each time instead of continuing on the existing curve, resulting in multiple curves.

![Screenshot 2021-05-15 at 11 40 47](https://user-images.githubusercontent.com/37438950/118357573-ba370800-b572-11eb-8c4d-dafc114aa80b.jpg)

### To Reproduce

Here is my code:

```python
while True:
  model.set_env(env)
  policy_kwargs = dict(
      features_extractor_class=CustomNet,
      features_extractor_kwargs=dict(features_dim=128),
  )
  model = PPO(
    "MlpPolicy", env, 
    policy_kwargs=policy_kwargs, verbose=1, 
    gamma=0.9995, device="cuda", 
    tensorboard_log="./env_1_logs/",
  )
  model.learn(total_timesteps=15000, tb_log_name="second_run", reset_num_timesteps=False)
  model.save(base_path + "/env_1")
```

### Expected behavior

Should continue the curve on the existing one (from the most recent timestep) instead of creating a new curve.

**Comment by araffin** [comment]
Hello,

>Here is my code:

The logic of your code is quite different from the logic presented in the example: you re-define the model multiple times instead of using the same one.

Probably a duplicate of https://github.com/DLR-RM/stable-baselines3/issues/339#issuecomment-790085919
(see my comment of why SB3 is behaving like that in that case)

To solve this issue completely, I think https://github.com/DLR-RM/stable-baselines3/issues/109 must be fixed first.

**Comment by araffin** [comment]
>As per the docs, passing the reset_num_timesteps=False should preserve the training curve in Tensorboard. Instead, what happens is that a new training curve is created with the same name each time:

This will be fixed in #469 where you can define you own logger.

**Comment by araffin** [comment]
Fixed by https://github.com/DLR-RM/stable-baselines3/pull/469, you should be able to do:

```python
import gym
from stable_baselines3 import PPO
from stable_baselines3.common.logger import configure

logger_path = "/tmp/sb3_log/"

# set up logger
new_logger = configure(logger_path, ["stdout", "tensorboard"])

env = gym.make("CartPole-v1")

model = PPO("MlpPolicy", env)
model.set_logger(new_logger)

model.learn(1000)

# Get env wrapped in a Monitor wrapper
env = model.get_env()

model.save(f"{logger_path}/ppo_cartpole.zip")
del model

# Load with the env that is in the same state when saving
# the PPO model
model = PPO.load(f"{logger_path}/ppo_cartpole.zip", env=env)
model.set_logger(new_logger)
model.learn(10000, reset_num_timesteps=False)


```
